test/e2e/snaps/test-snap-bip-32.spec.ts

```diff
@@ -26,7 +26,7 @@ describe('Test Snap bip-32', function () {
         // check for proper public key response using waitForSelector
         await driver.waitForSelector({
           css: '#bip32PublicKeyResult',
-          text: '"0x043e98d696ae15caef75fa8dd204a7c5c08d1272b2218ba3c20feeb4c691eec366606ece56791c361a2320e7fad8bcbb130f66d51c591fc39767ab2856e93f8dfb',
+          text: '"0x0425093606ebba8e4dd7b028fc1a7d77b79ef36fa231faf00c71123bac0e3dd7b7b608c7f5e532aafbff5519dd48b4d65c6b9ef6eb799c0bf9c683250448de214e',
```

Comment:
Why did these change for the primary SRP?

Reply:
The mocked state I used has a different primary SRP. Not sure if there's an easy way to create a mock state with the existing E2E SRP.

Reply:
Would prefer if we could do that as it makes it more convincing that everything works as intended 😄